docs/content/1.guide/1.configuration.md

```diff
@@ -40,81 +38,82 @@ nitro.timing=true
 
 ## Runtime Configuration
 
-Nitro provides a runtime config API to expose configuration within your application, with the ability to update it at runtime by setting environment variables.
-
-To expose config and environment variables to the rest of your app, you will need to define runtime configuration in your configuration file.
+Nitro provides a runtime config API to expose configuration within your application, with the ability to update it at runtime by setting environment variables. This is useful when you want to expose different configuration values for different environments (e.g. development, staging, production). For example, you can use this to expose different API endpoints for different environments or to expose different feature flags.
 
-**Example:**
+First, you need to define the runtime config in your configuration file.
 
 ::code-group
 ```ts [nitro.config.ts]
 export default defineNitroConfig({
   runtimeConfig: {
-    helloThere: "foobar",
+    foo: "bar", // `bar` is the default value
   }
 })
 ```
 
 ```ts [nuxt.config.ts]
 export default defineNuxtConfig({
   runtimeConfig: {
-    helloThere: "foobar",
```

Comment:
using `helloThere` were intentional to show multi part override via env (`NUXT_HELLO_THERE`) we might make it more realistic example like `apiToken` that still has two parts

Reply:
You are right. I've update it.